src/main.c

```diff
@@ -204,24 +213,70 @@ static const struct xdg_wm_base_listener xdg_wm_base_listener = {
 	.ping = xdg_wm_base_ping,
 };
 
-static void handle_dmabuf_format(void *data,
-		struct zwp_linux_dmabuf_v1 *zwp_linux_dmabuf, uint32_t format)
+static void handle_done(void* data, struct zwp_linux_dmabuf_feedback_v1* zwp_linux_dmabuf_feedback_v1)
```

Comment:
I'd rather call this something a bit more specific like `handle_dmabuf_formats_done`.